### ISSUE TITLE
Combining multiple trained models

### ISSUE THREAD

**Issue by clemyyyy** [MouseLand/cellpose#818]
Hello,

Thank you for the wonderful program, it is fantastic and will help my project. 

I seem to have a problem saving my own trained models. While training the same way as shown on your video and described in the documentation, Cellpose saves the models as different ones after each new image. I don't understand why that is.

Do you know why that might happen?

Kind regards.

**Comment by mrariden** [comment]
A new model is made each iteration through the HITL look. You can read more about this in the CP2 paper or in this issue: #723 . 

Closing for now, feel free to reopen.

**Comment by clemyyyy** [comment]
Hi there,

Thanks for getting back to me. I checked the thread and paper you mentioned. Unfortunately, it doesn't solve my issue. I observed that when training in the Cellpose GUI on my computer, the newly trained model is saved as a single, unified model. However, when I close and reopen the GUI later, the model gets divided into multiple models, all with the exact same names. The strange part is that the number of models it's split into matches the number of images I used to train the model. I'm a bit puzzled about what's happening.

Appreciate any insights you might have.
